### ISSUE TITLE
[Feature Request]Suffix appending behind of all class name

### ISSUE THREAD

**Issue by Hades32** [wuseal/JsonToKotlinClass#165]
Example json:

```json
{
"Master":{
  "a":1
  }
}
```

When using inner classes will generate a inner class and a field named `Mater`. 

```Kotlin
data class Blubber(
    val Master: Master?
) {
    data class Master(
        val a: Int?
    )
}
```

Easiest fix would be to append some postfix like `...Data`.

**Comment by kezhenxu94** [comment]
Hi @Hades32 I think making the first letter lowercase is better, what do you think? could you send a pull request to fix this? thanks



> Sent from [mini-github](https://github.com/kezhenxu94/mini-github)

**Comment by wuseal** [comment]
@Hades32 Try to select an annotation or use the suffix function:
![image](https://user-images.githubusercontent.com/9211902/56711416-4ba16a80-675d-11e9-9b55-6e0c193dc397.png)
*  By this method, all properties name will be renamed to lower camelcase, like ```val master: Master?```

![image](https://user-images.githubusercontent.com/9211902/56711434-5e1ba400-675d-11e9-938d-537954b1eff3.png)
* By this method, all properties name will append with a `Data` postfix, like ```val MasterData: Master?```

If there any problem, Please contact us, thanks.

**Comment by Hades32** [comment]
@wuseal both solution have issues: annotations make the code less portable and a suffix breaks compatibility with the actual json which is not under my control.

I'd prefer if there were an extension "Suffix after every class name". Then I'd just enter "Dto" and all the classes would be Dtos

**Comment by wuseal** [comment]
Yeah, An extension could implement your request feature, And the implemented code could be like this in extension instance class :
```kotlin

    override fun intercept(kotlinDataClass: KotlinDataClass): KotlinDataClass {

        val standTypes = listOf<String>("Int", "Double", "Long", "String", "Boolean")
        val originName = kotlinDataClass.name
        val newPropertyTypes =
            kotlinDataClass.properties.map { if (standTypes.contains(it.type)) it.type else it.type + "Dto" }

        val newProperties = kotlinDataClass.properties.mapIndexed { index, property ->

            val newType = newPropertyTypes[index]

            property.copy(type = newType)
        }
        return kotlinDataClass.copy(name = originName + "Dto", properties = newProperties)
    }
```
And If you interested in this, A PR for this feature will heighly appreciated.
You could refer to [Extensions](https://github.com/wuseal/JsonToKotlinClass/blob/master/doc_for_extensions.md) for building this feature.
For the code like upon, the code could be generated like this:
```kotlin
data class BlubberDto(
    val Master: MasterDto
)

data class MasterDto(
    val a: Int 
)
```

**Comment by smilechenbiao** [comment]
I have added this function,You can use it in later new versions.

**Comment by wuseal** [comment]
Fixed in 3.3.0-EAP-2
